interfaces/builtin/dbus.go

```diff
@@ -398,7 +425,17 @@ func (iface *DbusInterface) SanitizeSlot(slot *interfaces.Slot) error {
 	}
 
 	_, _, err := iface.getAttribs(slot.Attrs)
-	return err
+	if err != nil {
+		return err
+	}
+
+	// FIXME: also check that the dbus name is not already taken
+	//        by an existing snap
```

Comment:
How can we reliable implement this? Is there any way to query dbus for this as I think examining available configuration files isn't enough. org.freedesktop.DBus.ListActivatableNames looks like a good candidate for this. I am adding godbus to the snapd source tree already with https://github.com/snapcore/snapd/pull/3260 so would be fine to add it with this PR too.